src/pages/item/index.js

```diff
@@ -1,12 +1,59 @@
-import React from 'react';
+import React, { useContext, useState } from 'react';
+import '../../App.css'
+import './item.css'
+import {ItemContext} from '../../context/items'
+import {BasketContext} from '../../context/basket'
+import ItemList from '../home/items-list'
+import _ from 'lodash'
+import {useParams} from 'react-router-dom'
+import ItemTemplate from '../../components/item-template/item-template'
+
+
+
+ 
+
+const Item = (props) => {
+
+  const {updateBasket} = useContext(BasketContext)
+  const item = useContext(ItemContext)
+  
+  const itemPage = useContext(ItemContext)
+  const {id} = useParams()
+  const selection = _.find(ItemList, {id:  id} )
+  const OtherProduct = _.sampleSize(ItemList, 6)
+
+  
+  
+  
+  const [option, setOption] = useState (0)
 
 
-function App() {
   return (
-    <div style={{ textAlign: 'center', display: 'flex', justifyContent: 'center', height: '100vh', flexDirection: 'column' }}>
-      <p>item</p>
+    <div className='Container'>
+      <div className='Nav-Bar'>
+        <button className='Round-Button' onClick={() => props.history.push('/home')}>Back To Homepage</button>
+        <button className='Round-Button' onClick={() => props.history.push('/checkout')}>To Checkout</button>
+      </div>
+      <div className='Item-Display'>
+        <img className='product-image' src={selection.image.map(() =>  selection.image[option]) }/>
```

Comment:
Don't need the map here can just use the code you have inside the map function and it'll do the same thing!